test/others/libcriu/test_feature_check.c

```diff
@@ -0,0 +1,65 @@
+#include "criu.h"
```

Comment:
Would it be better to move `#include "criu.h"` after the standard headers?

Reply:
Better? Don't know. It should work in whichever way we include it. I guess this is just copied from the other test cases in this directory.